src/Files.Core.Storage/Contracts/IDeviceWatcher.cs

```diff
@@ -20,11 +20,6 @@ internal interface IDeviceWatcher
 		/// </summary>
 		event EventHandler<DeviceEventArgs>? ItemDeleted;
 
-		/// <summary>
-		/// Gets invoked when an item changing is detected by the watcher
-		/// </summary>
-		event EventHandler<DeviceEventArgs>? ItemChanged;
```

Comment:
Was this used for anything?

Reply:
I can't remember why added but I assume there was the same in WindowsDeviceWatcher class, which wraps 'ejected' and 'inserted' events of DeviceManager to 'changed' event.
But I think we shouldn't do that in the watcher and better do that in the service, taking which event it wants.

Reply:
Reverted.